reference/checkouts.sf.yml

```diff
@@ -1,4 +1,4 @@
-swagger: '2.0'
+swagger: '3.0'
```

Comment:
Should we update to 3.0 ? I saw many other files they use `openapi: 3.0.1`

When I tried to open it in the https://editor.swagger.io/ editor, it remind that: "Please indicate a valid Swagger or OpenAPI version field. Supported version fields are swagger: "2.0" and those that match openapi: 3.0.n (for example, openapi: 3.0.0)."

Also my VSCode Extension Swagger Preview cannot work for this change.

Reply:
@slsriehl @traci707 , It seems ` swagger: "2.0"` doesn't support `deprecated: true`, could we upgrade the document to `openapi: 3.0.1`?